src/jats/jats-exporter.ts

```diff
@@ -716,18 +756,14 @@ export class JATSExporter {
 
     // move ref-list from body to back
     back.appendChild(refList)
-
-    const bibliographyItems = this.models.filter(
-      hasObjectType<BibliographyItem>(ObjectTypes.BibliographyItem)
-    )
-
-    for (const bibliographyItem of bibliographyItems) {
+    const [meta] = this.citationProvider.makeBibliography()
+    for (const id of meta.entry_ids) {
+      const bibliographyItem = this.modelMap.get(id[0]) as BibliographyItem
```

Comment:
Is `id` an array? In the body editor it is treated as a `string`.

Reply:
yes, 
![image](https://github.com/Atypon-OpenSource/manuscripts-transform/assets/112621558/a7c62056-3a23-49a9-bc1a-909ce71132b1)

Reply:
That is... weird.